extensions/binding/org.eclipse.smarthome.binding.meteoblue/build.properties

```diff
@@ -0,0 +1,8 @@
+source..=src/main/java/
+output..=target/classes
+bin.includes=META-INF/,\
+             .,\
+             OSGI-INF/,\
+             ESH-INF/,\
+             NOTICE
+additional.bundles = org.eclipse.pde.ds.lib
```

Comment:
What is this about? Have never seen this so far...

Reply:
Not a clue. Removed.

Reply:
And after reviewing in Eclipse, I remember. Eclipse is issuing this warning after this line is removed:

````
DS Annotations missing from permanent build path
````

Reply:
None of the other bundles have this. So I guess you are not using the official IDE setup.
Please make sure that you have the "add DS Annotations to classpath" checked:

![screen shot 2018-09-26 at 11 25 58](https://user-images.githubusercontent.com/3244965/46070851-0beeb200-c17f-11e8-9411-1a9728527ad3.png)

Reply:
Actually it is the official setup. But it was set up 2-3 years ago.
I'll look into the preferences later. Thanks for the pointer.

Reply:
Just for the record, this setting was already enabled for both the project and the workspace. Doesn't help at all.